spec/models/group_invite_status_spec.rb

```diff
@@ -2,18 +2,21 @@
 
 require "rails_helper"
 
-RSpec.describe GroupInviteStatus, type: :model do
+RSpec.describe GroupInviteStatus, type: :invite_status do
   subject { GroupInviteStatus }
   let(:organization) { classroom_org }
   let(:grouping)     { create(:grouping, organization: organization) }
-  let(:group)        { Group.create(grouping: grouping, title: "Octokittens Team") }
+  let(:group)        { Group.create(grouping: grouping, title: "#{Faker::Company.name} Team") }
   let(:invitation)   { create(:group_assignment_invitation) }
 
   describe "valid", :vcr do
     let(:invite_status) do
       subject.create(group: group, group_assignment_invitation: invitation)
     end
 
+    # TODO: make Group factory so we can make GroupInviteStatus factory to test SetupStatus behavior with:
+    # it_behaves_like 'setup_status'
```

Comment:
We have some test duplication here because we are unable to use the shared `it_behaves_like` tests since we are don't have a Group FactoryBot. If we can make a Group FactoryBot then we could solve #1627